port: /dev/ttyUSB0 # USB1 or USB0, sometimes its 1
read_rate: 15
write_rate: 25
joints: {
    base: {id: 1, neutral: 205, max_angle: 180, min_angle: -60, max_speed: 30},
    shoulder: {id: 2, max_angle: -90, min_angle: 90, max_speed: 30},
    shoulder2: {id: 3, max_angle: 90, min_angle: -90, max_speed: 30},
    elbow: {id: 4, max_angle: -90, min_angle: 90, max_speed: 30},
    elbow2: {id: 5, max_angle: 90, min_angle: -90, max_speed: 30},
    wrist_flex: {id: 6, max_angle: 100, min_angle: -100, max_speed: 30},
    wrist_rotation: {id: 7, max_angle: 100, min_angle: -100, max_speed: 30},
    gripper: {id: 8, max_speed: 30},
}
controllers: {
    arm_controller: {type: follow_controller, joints: [base, shoulder, shoulder2, elbow, elbow2, wrist_flex, wrist_rotation], action_name: arm_controller/follow_joint_trajectory, onboard: False }
}
